src/_assets/css/_base.scss

```diff
@@ -41,3 +38,26 @@ dd {
 .material-icons {
   font: $site-font-icon;
 }
+
+.site-image-right {
```

Comment:
Is this a general purpose "site image" placement class, or has it been tailored to use in the get-started page?

Reply:
it's a general purpose class for any image that needs to be floated to the right of a paragraph.